client/package.json

```diff
@@ -60,6 +61,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.43.9",
+    "react-infinite-scroll-component": "github:techwithanirudh/react-infinite-scroll-component",
```

Comment:
I'm not sure that this will meet the stability requirements for this project, unless you're still working on this? might be best to change the PR back to a draft then

Reply:
Ok.